internal/backend/azure/config.go

```diff
@@ -22,7 +22,8 @@ type Config struct {
 	Container          string
 	Prefix             string
 
-	Connections uint `option:"connections" help:"set a limit for the number of concurrent connections (default: 5)"`
+	Connections uint   `option:"connections" help:"set a limit for the number of concurrent connections (default: 5)"`
+	AccessTier  string `option:"access-tier" help:"set the access tier for the blob storage (default: Hot)"`
```

Comment:
Isn't the default whatever value is configured for the bucket?

Reply:
Yes, the value would be inferred from the Storage Account. I have updated the documentation.